app.js

```diff
@@ -13,8 +14,9 @@ app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-// app.use(express.static(path.join(__dirname, "public")));
+app.use(express.static(path.join(__dirname, "public")));
 
 app.use("/api/milestones", milestonesRouter);
+app.use("/api/target", targetRouter);
```

Comment:
Can this be updated to `/api/targets`? This would make more sense and would be consistent with the `milestones` route.

Reply:
Updated.